src/main.ts

```diff
@@ -160,6 +160,7 @@ async function mergeResultBundle(
   const args = ['xcresulttool', 'merge']
     .concat(inputPaths)
     .concat(['--output-path', outputPath])
+    .concat(['--legacy'])
```

Comment:
Probably needs a version check added (i.e. Xcode 16 and above); thoughts on how to do that appreciated !